.storybook/config.js

```diff
@@ -81,6 +81,9 @@ window.googlesitekit.admin = window.googlesitekit.admin || googlesitekit.admin;
 window.googlesitekit.modules = window.googlesitekit.modules || googlesitekit.modules;
 window.googlesitekit.admin.assetsRoot = '/assets/';
 window.googlesitekit.isStorybook = true;
+window._googlesitekitBase = {
+	trackingUserOptInKey: 'wp_googlesitekit_tracking_optin',
+};
```

Comment:
I'd prefer if we kept this hard-coded in the JS code where it's used, similar to how it is now (just add the `wp_` prefix there).

Especially now that we've simplified the relevant code to only be in one place, we shouldn't introduce a variable for something as specific as this, especially as it's a constant value not depending on the setup in any way.

Reply:
Do you think we should add something like `blogPrefix` to the base data instead then?
Then perhaps we would have a utility for conditionally formatting user option meta keys with the prefix. What do you think?

Reply:
Good idea, I think that's more suitable to include in `_googlesitekitBase` because it might be needed elsewhere too.